pkg/tfbridge/info/info.go

```diff
@@ -233,6 +233,13 @@ type DocsPath struct {
 	TfToken string
 }
 
+type EditPhase int
+
+const (
+	PreCodeTranslation EditPhase = iota
+	PostCodeTranslation
```

Comment:
Any new object we export should have a comment explaining what it is *and* how to use it.
```suggestion
	// PreCodeTranslation directs a [DocsEdit] to occur before resource example code is translated.
	PreCodeTranslation EditPhase = iota
	// <useful comment here - links to DocsEdit>
	// TODO[<link appropriate issue>]: Right now, PostCodeTranslation is only called on installation docs.
	PostCodeTranslation
```